_events/10.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: event
+title:  "2020-02-11 Call"
+date:   2020-02-11 9:00
```

Comment:
Can we avoid requiring the same data to be entered twice?

Reply:
I think @ngehlenborg should weigh in here if there will be a title that is more than just the date of the call (that title was just the placeholder I put in).  I wanted to leave it as an option but could also see it as unnecessary.